jazz_apigee-proxy-aws/index.js

```diff
@@ -0,0 +1,101 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+const AWS = require('aws-sdk');
+const errorHandlerModule = require("./components/error-handler.js");
+const logger = require("./components/logger.js");
+const validation = require("./components/validation.js");
+
+module.exports.handler = (event, context, callback) => {
+  logger.init(event, context);
+  const errorHandler = errorHandlerModule();
+
+  validation.validateEvent(event, callback);
+
+  AWS.config.region = event.region;
+  const lambda = new AWS.Lambda();
+
+  //Defining the payload object
+  const payload = {
+    operation: event.operation,
```

Comment:
What does operation contain?

Reply:
Path suffix. Not required so removing it.